pkg/apis/vault/v1alpha1/vault_types_test.go

```diff
@@ -23,10 +23,10 @@ import (
 func TestGetVersion(t *testing.T) {
 	t.Run("Good", func(t *testing.T) {
 		tests := []string{
-			"hashicorp/vault:1.14.0",
+			"hashicorp/vault:1.14.1",
 			"hashicorp/vault:1.14",
-			"my.local.proxy/hashicorp/vault:1.14.0",
-			"my.local.proxy:5000/hashicorp/vault:1.14.0",
+			"my.local.proxy/hashicorp/vault:1.14.1",
+			"my.local.proxy:5000/hashicorp/vault:1.14.1",
```

Comment:
Updating this doesn't really make sense, although I understand why it might be a source of problem (grepping for `hashicorp/vault` will always yield this as a result).

Let's use some imaginary name instead of `hashicorp/vault` here, eg. `bank-vaults/my-vault:1.2.3` to make sure we don't keep updating this test.

Reply:
Good idea!